demo/src/app/confirm/confirm.component.ts

```diff
@@ -8,21 +8,20 @@ export interface ConfirmModel {
 
 @Component({
   selector: 'confirm',
-  template: `<div class="modal-dialog">
-                <div class="modal-content">
-                   <div class="modal-header">
-                     <button type="button" class="close" (click)="close()" >&times;</button>
-                     <h4 class="modal-title">{{title || 'Confirm'}}</h4>
-                   </div>
-                   <div class="modal-body">
-                     <p>{{message || 'Are you sure?'}}</p>
-                   </div>
-                   <div class="modal-footer">
-                     <button type="button" class="btn btn-primary" (click)="confirm()">OK</button>
-                     <button type="button" class="btn btn-default" (click)="cancel()">Cancel</button>
-                   </div>
-                 </div>
-                </div>`
+  template: `
+    <div class="modal_box">
+      <div class="modal_head">
```

Comment:
So the last thing i'll leave here for discussion. 

I guess i was thinking you would keep the bootstrap modal naming convention, which would leave it open to be used with BS3/BS4 but would supply a local css set if you wanted a pure version that would fill in some decent styles for those who didn't want to use Bootstrap. 

Perhaps we, a selling point here, is that we can implement any modal style 

Native - your styles and classnames
Bootstrap - using BS names as they were
Foundation - using reveal-modal setup 

Might be something we can explore to demonstrate the flexibility of the system? A demo of each.

Reply:
That's something we can easily do! Nice observation.

When I did this I was thinking more about using a BEM-like convention, instead of following one of these. 

About the failed task on Travis, to be honest, I never worked with CI, so I don't know exactly what may be the cause for this.

Reply:
The CI is calling 'ng lint' , you can do the same and fix the issues. Once that passes locally it will pass on Travis too

Reply:
@KevCJones I've sent you an e-mail, did you received it?

Reply:
No i didn't i'm afraid, i've only had mails via github from you

Reply:
Could you refork and then push this repo back up so i could even checkout the code my side to take a look? Will even fix the lints for you if you like

Reply:
if you change the package.json script for 'lint' should be `"lint": "tslint src/**/*.ts",` to support subfolders but only lint the typescript files.

Reply:
invited you to a slack channel for chats instead of email. You might see my username as outofskillerror